ein_erstes_projekt/ziel_hundert.py

```diff
@@ -19,19 +19,26 @@ def zahl_vom_spieler_holen():
     
     # solange nachfragen bis eine gültige Zahl eingegeben wird
     while True:
-        zahl_als_text = input("Wieviel möchten Sie addieren?")
+        zahl_als_text = input("Wieviel möchten Sie addieren?\n")
         zahl = int(zahl_als_text)
         if zahl < 1 or zahl > 10:
             print("Bitte eine Zahl von 1-10 eingeben")
         else:
             return zahl
 
 
-def zahl_waehlen():
+def zahl_waehlen(s):
     """Liefert die nächste Zahl, die der Computer addiert."""
     
     # TO DO: die KI nochmal überdenken!
-    return random.randint(1, 10)
+    if s % 11 is not 0:
+        r = -s + 11* int(s / 11 + 1) + 1
+    else:
+        r = -s + 11* int(s / 11)     + 1
```

Comment:
Falls ich das richtig verstehe, kann man das vereinfachen zu `r = 1`.
In diesem Zweig geht ja die Division glatt auf, d.h. `11*int(s/11)` ist einfach wieder `s`.
Das lässt auch die Logik gleich viel klarer werden: wenn Du eins von einer Gewinnzahl entfernt bist, zählst Du natürlich eins dazu.